docs/index.md

```diff
@@ -60,7 +60,7 @@ steampipe plugin install azuread
 | Item        | Description                                                                                                                                                                                                             |
 | ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Credentials | Use the `az login` command to setup your [Azure AD Default Connection](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli)                                                                               |
-| Permissions | Grant the following permissions to your user:<br />1. Application.Read.All<br />2. Policy.Read.ConditionalAccess<br />3. Directory.Read.All<br />4. Domain.Read.All<br />5. Group.Read.All<br />6. IdentityProvider.Read.All<br />7. User.Read.All                                                                                                                                                                   |
+| Permissions | Grant the following permissions to your user: <br /> <blockquote> <li>Application.Read.All </li><li>  Policy.Read.ConditionalAccess </li><li>  Directory.Read.All </li><li> Domain.Read.All </li><li> Group.Read.All </li><li> IdentityProvider.Read.All </li><li> User.Read.All  </blockquote>                                                                                                                                                                |
```

Comment:
@misraved Instead of having everything in `blockquote`, can you please have each item be wrapped as a code block, e.g., `Application.Read.All`? This is more common in our documentation.